spec/dummy/config/initializers/wrap_parameters.rb

```diff
@@ -1,14 +0,0 @@
-# Be sure to restart your server when you modify this file.
-
-# This file contains settings for ActionController::ParamsWrapper which
-# is enabled by default.
-
-# Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
-ActiveSupport.on_load(:action_controller) do
-  wrap_parameters format: [:json]
-end
-
-# To enable root element in JSON for ActiveRecord objects.
-# ActiveSupport.on_load(:active_record) do
-#   self.include_root_in_json = true
-# end
```

Comment:
This file had to be removed because it tries to wrap the JSON parameters into an object based on the controller name. In our case we are expecting the openapi-parser to do the validation for us and it doesn't like the wrapped object which is an extra parameter and it rejects it. None of our other apps catalog, approval, sources and topology are using the wrap parameters feature of rails.